applications/asset_tracker_v2/src/modules/Kconfig.gnss_module

```diff
@@ -5,7 +5,7 @@
 #
 
 menuconfig GNSS_MODULE
-	bool "GPS module"
+	bool "GNSS module"
```

Comment:
@jtguggedal I believe this was missed in #6392

Reply:
Yep, looks like it